recipes/ade/all/conanfile.py

```diff
@@ -0,0 +1,69 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class AdeConan(ConanFile):
+    name = "ade"
+    license = "Apache-2.0"
+    homepage = "https://github.com/opencv/ade"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Graph construction, manipulation, and processing framework"
+    topics = ("graphs", "opencv")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+    }
+
+    _cmake = None
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def validate(self):
+        if self.settings.compiler.get_safe("cppstd"):
+            tools.check_min_cppstd(self, 11)
+        if self.settings.compiler.libcxx != "libstdc++11":
+            raise ConanInvalidConfiguration("libstdc++11 required")
```

Comment:
Why is that?
The only difference between libstdc++ and libstdc++11 is in the abi.